tests/integ/ml/test_transformer.py

```diff
@@ -0,0 +1,524 @@
+        [[6, 4, 5], [2, 4, 5], [2, 3, 6]], schema=["a", "b", "c"]
+    )
+    res = model.transform(df_unknown_value)
+    Utils.check_answer(
+        res,
+        [
+            Row(
+                A=2,
+                B=4,
+                C=5,
+                ONEHOTARRAY="[\n  6,\n  [\n    1,\n    2,\n    4\n  ],\n  1\n]",
+            ),
+            Row(A=6, B=4, C=5, ONEHOTARRAY="[\n  6,\n  [\n    2,\n    4\n  ],\n  1\n]"),
+            Row(
+                A=2,
+                B=3,
+                C=6,
+                ONEHOTARRAY="[\n  6,\n  [\n    1,\n    3,\n    5\n  ],\n  1\n]",
+            ),
+        ],
+    )
+
+
+def test_merge_ordinalcoder(session):
+    df = session.create_dataframe(
+        [[1, 4, 5], [2, 4, 5], [2, 3, 6]], schema=["a", "b", "c"]
+    )
+    ordinalcoder = OrdinalEncoder()
+    ordinalcoder.input_cols = ["A", "B", "C"]
+    ordinalcoder.output_cols = ["ORDINALARRAY"]
+    model = ordinalcoder.fit(df)
+
+    # test category table
+    df_table = session.table(model._category_table_name["A"])
+    Utils.check_answer(
+        df_table,
+        [Row(STATES_A=1, STATES_A_ENCODER=0), Row(STATES_A=2, STATES_A_ENCODER=1)],
+    )
+
+    res = model.transform(df)
+    Utils.check_answer(
+        res,
+        [
+            Row(A=1, B=4, C=5, ORDINALARRAY="[\n  0,\n  0,\n  0\n]"),
+            Row(A=2, B=4, C=5, ORDINALARRAY="[\n  1,\n  0,\n  0\n]"),
+            Row(A=2, B=3, C=6, ORDINALARRAY="[\n  1,\n  1,\n  1\n]"),
+        ],
+    )
+
+
+def test_preset_category_ordinal_encoder(session):
+    df = session.create_dataframe(
+        [[1, 4, 5], [2, 4, 5], [2, 3, 6]], schema=["a", "b", "c"]
+    )
+    df_unknown = session.create_dataframe(
+        [[3, 4, 5], [2, 4, 5], [2, 3, 6]], schema=["a", "b", "c"]
+    )
+    ordinalencoder = OrdinalEncoder(category=[["1", "2"], ["4", "3"], ["5", "6"]])
+    ordinalencoder.input_cols = ["A", "B", "C"]
+    ordinalencoder.output_cols = ["ONEHOTARRAY"]
+    model = ordinalencoder.fit(df)
+
+    # test category table
+    df_table = session.table(model._category_table_name["A"])
+    Utils.check_answer(
+        df_table,
+        [Row(STATES_A="1", STATES_A_ENCODER=0), Row(STATES_A="2", STATES_A_ENCODER=1)],
+    )
+
+    res = model.transform(df)
+    Utils.check_answer(
+        res,
+        [
+            Row(A=1, B=4, C=5, ORDINALARRAY="[\n  0,\n  0,\n  0\n]"),
+            Row(A=2, B=4, C=5, ORDINALARRAY="[\n  1,\n  0,\n  0\n]"),
+            Row(A=2, B=3, C=6, ORDINALARRAY="[\n  1,\n  1,\n  1\n]"),
+        ],
+    )
+
+    # test when there are unknown category in given df
+    res = model.transform(df_unknown)
+    Utils.check_answer(
+        res,
+        [
+            Row(A=3, B=4, C=5, ONEHOTARRAY="[\n  undefined,\n  0,\n  0\n]"),
+            Row(A=2, B=3, C=6, ONEHOTARRAY="[\n  1,\n  1,\n  1\n]"),
+            Row(A=2, B=4, C=5, ONEHOTARRAY="[\n  1,\n  0,\n  0\n]"),
+        ],
+    )
+
+
+def test_binarizer(session):
```

Comment:
Probably add one more test case about float data and float threshold (you can also try to use nan as threshold)